frontend/src/components/mydashboard/MydashboardItemComponent.js

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import { Column } from 'simple-flexbox';
```

Comment:
Please Use Col instead and don't use this one if possible. simple-flexbox and react-grid-system are not best friends! @IremToroslu @manifau